recipes/refseq_masher/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "refseq_masher" %}
+{% set version = "0.1.0" %}
+{% set sha256sum = "3d4460966f5a7994b66c2c87cfe5a0116d8b2bdf6a9b8b0e4ea57cd891e26708" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: {{ sha256sum }}
+
+build:
+  skip: True # [py27]
+  skip: True # [not linux]
```

Comment:
Can you remove that in favor of altered test cases (see below).

Reply:
Thanks! Removed in 898b3bd8b015425cdf81b3e5e6d921e302cf06ad